tests/rollapp_genesis_event_test.go

```diff
@@ -215,4 +214,26 @@ func TestRollappGenesisEvent_EVM(t *testing.T) {
 	require.NoError(t, err)
 
 	testutil.AssertBalance(t, ctx, rollapp1, escrowAddress, rollapp1.Config().Denom, genesisCoin.Amount)
+
+	denommetadata, err := dymension.GetNode().QueryDenomMetadata(ctx, genesisCoin.Denom)
```

Comment:
@trinitys7 `vfc` created by hook should be tested here as well? or a separate test?

Reply:
I think it's kinda hard for e2e-test to query a Solidity contract. Do you have any idea about how we do that?

Reply:
I thought `vfc` has a query command, but apparently it doesn't
if u have access to the `x/evm` keeper, u can use `k.GetVirtualFrontierBankContractAddressByDenom`

Reply:
added separate issue for it https://github.com/dymensionxyz/e2e-tests/issues/71